ani.ter-markosyan/repo/Lesson_3/exercise_21/exercise_21.py

```diff
@@ -0,0 +1,5 @@
+number = int(input("Enter a number: "))
+if (number % 2 == 0):
```

Comment:
RW: No parenthesis needed for comparison operator in python. Please use python syntax.